_posts/2020-07-14-newpipe-0.19.6-release.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "NewPipe 0.19.6 released: Audio/video desync bug and search improvements"
+short: "NewPipe 0.19.6 released"
+date: 2020-07-14 00:00:00 +0100
+categories: [release, pinned]
+author: gkeegan
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+No more unsynced videos! This release will let your videos make sense and improve searching in NewPipe to make it more reliable and widespread.
+
+<!-- more -->
+
+## Additions
+
+You can now add a playlist directly to your main page as a tab, a great change for those who frequently use NewPipe for music. <sup>[@Royosef in #3506](https://github.com/TeamNewPipe/NewPipe/pull/3506)</sup>
+
+There is an option to prevent subscriptions from being visible in the subscription picker for feeds which will be helpful to sort your subscriptions into different categories. <sup>[@mauriciocolli in #3404](https://github.com/TeamNewPipe/NewPipe/pull/3404)</sup>
+
+Speaking (writing) of the subscription picker, it now has a search bar so you can easily find what you are looking for. <sup>[@mauriciocolli in #3373](https://github.com/TeamNewPipe/NewPipe/pull/3373)</sup>
+
+More search changes include "did you mean" & "showing result for" suggestions from YouTube so that you don't have to restart a search if you make a tpyo. <sup>[@Royosef in #3471](https://github.com/TeamNewPipe/NewPipe/pull/3471)</sup>
```

Comment:
It's not YouTube-only and @B0pol made the appropriate changes on the NPE side

Reply:
```suggestion
More search changes include "did you mean" & "showing result for" suggestions so that you don't have to restart a search if you make a tpyo. <sup>[@B0pol in Extractor #311](https://github.com/TeamNewPipe/NewPipeExtractor/pull/311) and [@Royosef in #3471](https://github.com/TeamNewPipe/NewPipe/pull/3471)</sup>
```